contracts/Archetype.sol

```diff
@@ -675,14 +688,21 @@ contract Archetype is
       revert MintNotYetStarted();
     }
 
-    if (i.limit < config.maxSupply) {
+    if (i.limit < i.maxSupply) {
       uint256 totalAfterMint = _minted[msg.sender][auth.key] + quantity;
 
       if (totalAfterMint > i.limit) {
         revert NumberOfMintsExceeded();
       }
     }
 
+    if (i.maxSupply < config.maxSupply) {
+      uint256 totalAfterMint = listSupply[auth.key] + quantity;
+      if (totalAfterMint > i.limit) {
```

Comment:
Shouldn't this be `totalAfterMint > i.maxSupply`?

Reply:
oh wow yeah thank you for catching that, surprised I missed this